### DISCUSSION TITLE
Getting a license

### DISCUSSION THREAD

**Discussion by asmith555** [nager/Nager.Date#426] [General]
Is there any way to get a license for the newer versions?

**Comment by tinohager** [comment]
Yes, with a GitHub sponsorship for nager you automatically receive the license key

**Comment by gayancc** [comment]
Hi I did sponsored the project but didn't receive the license key and i cannot find it on sponsor page

**Reply by tinohager** [reply]
I have send you an email :)

**Reply by tinohager** [reply]
@gayancc Did you receive my email?
